### ISSUE TITLE
TypeScript errors in newly created app

### ISSUE THREAD

**Issue by mikkom** [wmonk/create-react-app-typescript#41]
I get the following error when creating a new app with react-scripts-ts:

```
➜  uname -a
Darwin Mikkos-MBP-2.Home 16.4.0 Darwin Kernel Version 16.4.0: Thu Dec 22 22:53:21 PST 2016; root:xnu-3789.41.3~3/RELEASE_X86_64 x86_64
➜  node --version
v6.10.0
➜  yarn --version
0.21.3
➜  create-react-app --version
1.3.0
➜  create-react-app my-app --scripts-version=react-scripts-ts
...
➜  cd my-app
➜  yarn run build
yarn run v0.21.3
$ react-scripts-ts build
Creating an optimized production build...
ts-loader: Using typescript@2.3.0 and /Users/mikkom/prj/tmp/my-app/tsconfig.json
Failed to compile.

Error in /Users/mikkom/prj/tmp/my-app/src/index.tsx
(7,3): error TS2322: Type '{}' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<App> & Readonly<{ children?: ReactNode; }> & null'.
  Type '{}' is not assignable to type 'null'.

Error in /Users/mikkom/prj/tmp/my-app/src/App.test.tsx
(7,19): error TS2322: Type '{}' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<App> & Readonly<{ children?: ReactNode; }> & null'.
  Type '{}' is not assignable to type 'null'.

 4 |
 5 | it('renders without crashing', <span class="hljs-function">() =></span> {
 6 |   const div = document.createElement('div');
 7 |   ReactDOM.render(<App />, div);
-----------------------^
 8 | });
 9 |

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by onyno** [comment]
I had the same error, replaced the first null with an empty property literal in App.tsx

`class App extends React.Component<{}, null> {`

**Comment by wmonk** [comment]
Hey, yes this is an error with the newly release React 15.5.x. I will update

**Comment by wmonk** [comment]
Fixed in 1.9.1

**Comment by mikkom** [comment]
Thanks, seems to be working great!
